modules/ensemble/lib/framework/apiproviders/firestore/firestore_app.dart

```diff
@@ -116,6 +116,23 @@ class FirestoreApp {
         query = query.limitToLast(queryMap['limitToLast']);
       }
     }
+    if (api.containsKey('aggregate')) {
+        String aggregateType = api['aggregate']['type'];
```

Comment:
@TheNoumanDev firestore supports aggregates over multiple fields. Your schema and PR code is only allowing one field, why is that?